charts/kyverno-notation-aws/templates/_helpers/_labels.tpl

```diff
@@ -26,6 +26,12 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/version: {{ template "kyverno-notation-aws.chartVersion" . }}
 {{- end -}}
 
+{{- define "kyverno-notation-aws.labels.values" -}}
```

Comment:
If we're trying to add custom labels throughout the chart's resources, maybe having the labels included inside the existing `kyverno-notation-aws.labels.common` would be a bit better 🤔 ? What do you think ?

```
{{- define "kyverno-notation-aws.labels.common" -}}
{{- template "kyverno-notation-aws.labels.merge" (list
 (include "kyverno-notation-aws.labels.helm" .)
 (include "kyverno-notation-aws.labels.version" .)
 (toYaml .Values.customLabels)
) -}}
{{- end -}}
```

Reply:
Much better! Updated!